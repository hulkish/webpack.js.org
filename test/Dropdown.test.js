import React from 'react';
import Dropdown from '../components/Dropdown/Dropdown';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const output = renderer.create(
    <Dropdown
      items={[
        { title: 'English', url: 'https://webpack.js.org/' },
        { title: '中文', url: 'https://doc.webpack-china.org/' }
      ]}
    />
  ).toJSON();

  expect(output).toMatchSnapshot();
});
