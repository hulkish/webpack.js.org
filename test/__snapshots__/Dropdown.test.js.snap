// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="dropdown "
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
>
  <img
    className="dropdown__language"
    src={Object {}}
  />
  <i
    className="dropdown__arrow"
  />
  <div
    className="dropdown__list "
  >
    <ul>
      <li>
        <a
          href="https://webpack.js.org/"
        >
          <span>
            English
          </span>
        </a>
      </li>
      <li>
        <a
          href="https://doc.webpack-china.org/"
        >
          <span>
            中文
          </span>
        </a>
      </li>
    </ul>
  </div>
</div>
`;
